src/CMakeLists.txt

```diff
@@ -14,6 +14,22 @@ add_custom_command(
   WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
   DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/gen_types.py)
 
+add_custom_command(
```

Comment:
Why do you continue to keep them in the repo if you regenete them automatically? I would prefer to remove them from the repo.